pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -123,6 +123,23 @@ func (builder *nodeEntityDTOBuilder) BuildEntityDTOs(nodes []*api.Node, nodesPod
 			glog.Errorf("Failed to get node properties: %s", err)
 			nodeActive = false
 		}
+		for _, property := range properties{
+			var nodeType string
+			spot := strings.Contains(property.GetName(), util.EKSCapacityType) && property.GetValue() == util.EKSSpot
+			windows := strings.Contains(property.GetName(), util.NodeLabelOS) && property.GetValue() == util.WindowsOS
+			if spot || windows {
+				if spot{
+					nodeType = util.EKSSpot
+				} else if windows {
+					nodeType = util.WindowsOS
+				} else {
+					nodeType = "unknown"
+				}
+ 				glog.Warningf("Node %s is a %s type, setting to not provisionable and not suspendable.", node.GetName(), nodeType)
```

Comment:
This should be a v2 Info level message. Just say something like
`Suspend and provision is disabled for node that is either AWS EC2 spot instance or node with Windows OS.`

Reply:
Updated the message, thanks